boards/arm/ubx_evkninab3_nrf52840/ubx_evkninab3_nrf52840.dts

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright (c) 2021 u-blox AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "u-blox EVK NINA-B3 nRF52840";
+	compatible = "nordic,ubx_evkninab3_nrf52840";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+			label = "Red LED 0";
+		};
+		led1: led_1 {
+			gpios = <&gpio1 0 GPIO_ACTIVE_LOW>;
+			label = "Blue LED 1";
+		};
+		led2: led_2 {
+			gpios = <&gpio0 25 GPIO_ACTIVE_LOW>;
+			label = "Green LED 2";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		pwm_led0: pwm_led_0 {
+			pwms = <&pwm0 13>;
+		};
+		pwm_led1: pwm_led_1 {
+			pwms = <&pwm0 32>;
+		};
+		pwm_led2: pwm_led_2 {
+			pwms = <&pwm0 25>;
+		};
+		red_pwm_led: red_pwm_led {
```

Comment:
Why do you define these LEDs twice? Wouldn't it be sufficient to use only aliases, e.g.:
```
red-pwm-led = &pwm_led_0;
```

Reply:
Done - thanks for the suggestion.